spec/lib/reporting_spec.rb

```diff
@@ -93,17 +82,21 @@
         expect(user1.notes.count).to eq(2)
         expect(reporting.users[:with_notes]).to eq(1)
         expect(reporting.users[:with_notes_percentage]).to eq(100.0)
+        expect(reporting.users[:without_notes]).to eq(0)
+        expect(reporting.users[:without_notes_percentage]).to eq(0)
       end
     end
 
     context 'when 1 user has notes and 1 has none' do
-      it 'notes count is 1 and percentage is 50' do
+      it 'with_notes is 1, without notes is 1 and percentage is 50' do
```

Comment:
Can we make this any terser and BA friendly? For example:

`when half the users create notes`, it `is 50%`